src/precice/SolverInterface.hpp

```diff
@@ -89,17 +89,14 @@ class PRECICE_API SolverInterface {
    * - Sets up a connection to the other participants of the coupled simulation.
    * - Creates all meshes, solver meshes need to be submitted before.
    * - Receives first coupling data. The starting values for coupling data are zero by default.
-   * - Determines length of the first timestep to be computed.
```

Comment:
This is still true

Reply:
Fixed d1a8462da65245df82017a21710afd5d51934a03